core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -160,17 +160,16 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     } else false
   }
 
-  private def aclMatch(session: Session, operations: Operation, resource: Resource, principal: KafkaPrincipal, host: String, permissionType: PermissionType, acls: Set[Acl]): Boolean = {
+  private def aclMatch(session: Session, operations: Operation, resource: Resource, principal: KafkaPrincipal, host: String, permissionType: PermissionType, acls: Set[Acl]): Boolean =
     acls.find ( acl =>
```

Comment:
Sorry, I was trying to suggest replacing the parenthesis with braces for consistency (somewhat annoyingly, you have to move the `&&` to the previous lines to make it work). It's not a big deal, so if it doesn't actually seem better, feel free to ignore.